status/backend.json

```
@@ -71,8 +71,8 @@
     "newPaymentSection": {
       "enabled": false,
       "min_app_version": {
-        "ios": "3.0.0.0",
-        "android": "3.0.0.0"
+        "ios": "2.64.0.1",
+        "android": "2.64.0.1"
```

```suggestion
        "ios": "2.65.0.0",
        "android": "2.65.0.0"
```

Since the 2.65.0.0 includes a fix on the 2.64.0 release, what about setting up only the newer version?

---

It is still an open point from the "top managers", this PR is a placeholder in order to wait their decisions about that